README.md

```diff
@@ -226,7 +230,7 @@ Congratulations! You now have a failing spec to develop against.
 
 At this stage, you'll want to be able to run your specs one at a time while you implement each feature. At the bottom of the failed pact:verify output you will see the commands to rerun each failed interaction individually. A command to run just one interaction will look like this:
 
-    $ rake pact:verify PACT_DESCRIPTION="a request for something" PACT_PROVIDER_STATE="something exists"
+    $ rake pact:verify PACT_DESCRIPTION="a request for a alligator" PACT_PROVIDER_STATE="alligator exists"
 
```

Comment:
$ rake pact:verify PACT_DESCRIPTION="a request for an alligator" PACT_PROVIDER_STATE="an alligator exists"